addons/point_of_sale/static/tests/tours/helpers/ProductScreenTourMethods.js

```diff
@@ -248,6 +272,13 @@ odoo.define('point_of_sale.tour.ProductScreenTourMethods', function (require) {
             }
             return steps;
         }
+        addCustomerNote(note) {
+            const res = [];
+            res.push(...this._do.clickOrderlineCustomerNoteButton());
```

Comment:
don't you need here to do something like: TextAreaPopup.check.isShown();

Reply:
Not needed